canonical-websites/www.ubuntu.com
# Minify and combine js

## Done

- This work is to reduce the number of scripts loaded as well as minifying the JS.
- Add Webpack and Babel.
- Moved some files to modules where needed.
- Removed global nav/cookie policy script copying.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that all Javascript works as before, the following covers instances of all bundled/minified files:
  - Check global nav, cookie policy, main nav dropdowns etc. on all pages
  - Check charts are shown on /about/release-cycle
  - Check charts and sticky nav work on /desktop/statistics
  - Check the contact us button on the bottom of pages like /openstack/consulting opens the form
  - Check the country select box works on /openstack/contact-us?product=foundation-cloud


## Issue / Card

- Fixes: #4773
